setup.py

```diff
@@ -0,0 +1,31 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="fpoptest",
```

Comment:
Do we want to use this name?

Reply:
I change the name to fpop.But I have not submitted to pypi.